src/app/acesso/cadastro/endereco-cadastro/endereco-cadastro.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ConsultaEstadosService, Estado, Cidade } from 'src/app/servicos/consulta-estados.service';
+import { Endereco } from 'src/app/interfaces/endereco';
+import { enderecoFormControl } from 'src/app/util/form-controls';
```

Comment:
Codacy found an issue: ['enderecoFormControl' is defined but never used.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)